src/examples/iris/helper.cpp

```diff
@@ -26,10 +26,10 @@ void readIrisData(const std::string fileName,
   }
   std::string line;
   std::string value;
-  while(marian::io::getline(in, line)) {
+  while(std::getline(in, line)) {
```

Comment:
Wrong approach, please rather use InputFileStream and io::getline. I would like to discourage using std::ifstream and std::getline, so we can intercept this.